invenio_sipstore/models.py

```diff
@@ -71,24 +65,35 @@ class SIP(db.Model, Timestamp):
     agent = db.Column(JSONType, default=lambda: dict(), nullable=False)
     """Agent information regarding given SIP."""
 
+    archivable = db.Column(
+        db.Boolean(name='ck_sipstore_archivable'),
```

Comment:
should we follow simple naming as in here: https://github.com/inveniosoftware/invenio-files-rest/blob/master/invenio_files_rest/models.py#L234